app/src/main/java/asd/org/ahorcado/activities/MainActivity.java

```diff
@@ -1,60 +1,117 @@
 /**
  * Muber 2016. Copyright © All rights reserved.
  */
-
 package asd.org.ahorcado.activities;
 
-import android.content.Intent;
+import android.graphics.Point;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
+import android.view.Display;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
+import android.widget.Button;
+import android.widget.TableLayout;
+import android.widget.Toast;
 
 import asd.org.ahorcado.R;
+import asd.org.ahorcado.controller.GameController;
+import asd.org.ahorcado.exceptions.LostLifeException;
+import asd.org.ahorcado.fragments.WordFragment;
 
 public class MainActivity extends AppCompatActivity {
 
+    private GameController gameController;
+
+    private static int INTERVAL = 2000; //2 second
+
+    private long firstClickTime;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
-
+        TableLayout tl = (TableLayout) this.findViewById(R.id.tableLayout);
+        tl.setVisibility(View.INVISIBLE);
+        gameController = new GameController(this);
     }
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        // Inflate the menu; this adds items to the action bar if it is present.
         getMenuInflater().inflate(R.menu.menu_main, menu);
         return true;
     }
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        // Handle action bar item clicks here. The action bar will
-        // automatically handle clicks on the Home/Up button, so long
-        // as you specify a parent activity in AndroidManifest.xml.
         int id = item.getItemId();
-
-        //noinspection SimplifiableIfStatement
         if (id == R.id.action_settings) {
             return true;
         }
-
         return super.onOptionsItemSelected(item);
     }
 
+    public void lunchGame(View view) {
+        gameController.newMatch();
+        Button button = (Button) findViewById(R.id.play_game);
+        button.setVisibility(View.INVISIBLE);
+        TableLayout tl = (TableLayout) this.findViewById(R.id.tableLayout);
+        tl.setVisibility(View.VISIBLE);
+        WordFragment f = (WordFragment) getFragmentManager().findFragmentById(R.id.WordFragment);
+        f.updateImage(gameController.obtainPartialWord(), widthDisplay(), heightDisplay());
+        try {
+            UtilActivity.setBackground(view, gameController.getRemainingLives());
+        } catch (LostLifeException e) {
+        }
+    }
+
     @Override
     public void onBackPressed() {
-        finish();
-        System.exit(0);
+        if (firstClickTime + INTERVAL > System.currentTimeMillis()) {
+            super.onBackPressed();
+            finish();
+            System.exit(0);
```

Comment:
Yes, it's necessary because the method finish() kill this activity and System.exit(0) kill the App in 0 milliseconds.